src/psyclone/psyir/transformations/omp_loop_trans.py

```diff
@@ -266,4 +274,4 @@ def apply(self, node, options=None):
                     "nthreads", tag="omp_num_threads",
                     symbol_type=DataSymbol, datatype=INTEGER_TYPE)
 
-        super().apply(node, options)
+        super().apply(node, options, **kwargs)
```

Comment:
Can we merge `options` into `**kwargs` now (allowing for the `-` -> `_` replacement)?

Reply:
I tried it here and its possible I think - I think some function to do this would be good so it can be used in more places, I suspect as a class function of Transformation?